.github/workflows/backend-build-test.yml

```
@@ -0,0 +1,26 @@
+name: Backend build and test
+
+on:
+  push:
+    paths:
+      - "backend/**"
+
+jobs:
+  build-test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Setup .NET
+        uses: actions/setup-dotnet@v4
+        with:
+          dotnet-version: '8.0.x'
```

Small nitpicking, but looks cleaner if the quotations is consequentially used as singular og double 😛
```suggestion
          dotnet-version: "8.0.x"
```

---

Hard agree!